test/integration/supervisor_login_test.go

```diff
@@ -2403,7 +2812,12 @@ func wantGroupsInAdditionalClaimsIfGroupsExist(additionalClaims map[string]any,
 	return additionalClaims
 }
 
-func requireSuccessfulLDAPIdentityProviderConditions(t *testing.T, ldapIDP *idpv1alpha1.LDAPIdentityProvider, expectedLDAPConnectionValidMessage string) {
+func requireSuccessfulLDAPIdentityProviderConditions(
```

Comment:
This function and the 3 functions below it are all very similar. Is there any way to refactor these?

Reply:
will be done in next push